test/Extensions/Tester.Cassandra/Clustering/CassandraClusteringTableTests.cs

```diff
@@ -100,6 +100,29 @@ public async Task MembershipTable_InsertRow()
         Assert.Single(data.Members);
     }
 
+    [Fact]
+    public async Task MembershipTable_DefaultTtl_InsertRow()
+    {
+        var (membershipTable, _) = await CreateNewMembershipTableAsync(defaultTtlSeconds:600);
```

Comment:
Can a test be added with a short (1 sec) TTL and then wait 5 sec to see that the row is gone?
Does the cleanup in Cassandra work at that resolution?

Reply:
Yes, added!